.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: '[Bug]: '
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Thanks for opening a bug report!
+Before hitting the button, please fill in as much of the template below as you can.
+When describing the steps to reproduce the bug,
+try following your own instructions from scratch and make sure everything works as
+you expect.
+Be ready for follow-up questions, and please respond in a timely manner.
+If we can't reproduce a bug we might close your issue.
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2. 
+3. 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear and concise description of what actually happened.
+
+**Environment:**
+ - Kadalu Version: [e.g., 0.8.6]
```

Comment:
- a comment about getting the version here would be helpful, something along the lines
```
<!--
Output from one of commands
1. If kubectl-kadalu plugin is installed
$ kubectl kadalu version

2. If plugin isn't installed
$ kubectl exec -it deploy/operator -n kadalu -- bash -c 'kubectl-kadalu version'
-->
```